oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -1,35 +1,48 @@
-${PYTHON_SHEBANG}
-from __future__ import absolute_import, division, print_function
+#!/usr/bin/env python3
 import subprocess
 import sys
 import xml.etree.ElementTree as ET
+from pathlib import Path
 
-if sys.version_info < (3,0):
-    import ConfigParser
-else:
-    import configparser
+import configparser
+
+
+api_config_file = '/etc/columnstore/cmapi_server.conf'
+
+
+def get_key():
+    cmapi_config = configparser.ConfigParser()
+    try:
+        cmapi_config.read(api_config_file)
+    except FileNotFoundError:
+        return ''
+
+    if 'Authentication' not in cmapi_config.sections():
+        return ''
+    return cmapi_config['Authentication'].get('x-api-key', '')
+
+
+def get_version():
+    return '0.4.0'
```

Comment:
Are we always going to have a literal or are we going to find a way to dynamically get this information. This one is bit tricky since cluster agent is going to be packaged separately.

Reply:
Yes, hardcoded version is a planned move b/c this protects us from a version mismatch.